source/strings.tex

```diff
@@ -4661,8 +4661,8 @@
 
 \pnum
 \begin{note}
-The library provides implicit conversions from \tcode{const charT*} and \tcode{std::basic_string<charT, ...>} to \tcode{std::basic_string_view<charT, ...>} so that user code can accept just \tcode{std::basic_string_view<charT>} as a non-templated parameter wherever a sequence of characters is expected.
-User-defined types should define their own implicit conversions to \tcode{std::basic_string_view} in order to interoperate with these functions.
+The library provides implicit conversions from \tcode{const charT*} and \tcode{basic_string<charT, ...>} to \tcode{basic_string_view<charT, ...>} so that user code can accept just \tcode{basic_string_view<charT>} as a non-templated parameter wherever a sequence of characters is expected.
+User-defined types should define their own implicit conversions to \tcode{basic_string_view} in order to interoperate with these functions.
 \end{note}
```

Comment:
Let me close this PR in the interest of not diluting attention too much for what are basically subjective issues. Feel free to reopen if you feel really strongly, but in that case I'd leave this for the project editor to process.